### PR TITLE
GrafanaReportPanelReportSource data model

### PR DESCRIPTION
**Pull request by speezepearson**

### CODE REVIEW

File: conf/db/migration/metrics_portal_ddl/V18__grafana_report_source.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/**
```
Comment (vjkoskela) [inline conf/db/migration/metrics_portal_ddl/V18__grafana_report_source.sql]: I think this should be `/*` not `/**`.

File: conf/db/migration/metrics_portal_ddl/V18__grafana_report_source.sql [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ALTER TABLE portal.report_sources ADD COLUMN time_range_period VARCHAR(255) NOT NULL;
+ALTER TABLE portal.report_sources ADD COLUMN time_range_width_periods INTEGER;
+ALTER TABLE portal.report_sources ADD COLUMN time_range_end_periods_ago INTEGER;
```
Comment (vjkoskela) [inline conf/db/migration/metrics_portal_ddl/V18__grafana_report_source.sql]: This implies the end is in the past?

**Comment by vjkoskela** [comment]
Minor changes plus whatever the build is complaining about. Looks good.

File: conf/db/migration/metrics_portal_ddl/V18__grafana_report_source.sql (hunk 1 above)
Comment (speezepearson) [reply conf/db/migration/metrics_portal_ddl/V18__grafana_report_source.sql]: Hmm, all the other `.sql` files in this directory use `/**`.

**Pull request merged**